inspirehep/modules/workflows/workflows/article.py

```diff
@@ -280,7 +273,15 @@
     IF_ELSE(
         is_marked('is-update'),
         [
-            mark('approved', True)
+            merge_articles,
+            IF(
+                has_conflicts,
+                halt_record(
+                    action='merge_approval',
+                    message='Submission halted for merging conflicts.'
+                ),
+            ),
+            mark('approved', True),
```

Comment:
`mark('merged', True)` :muscle:

Reply:
this will add the mark even when the ``merge_articles`` is disabled by the feature flag :/